azext_edge/edge/providers/orchestration/resources/connector/opcua/certs.py

```diff
@@ -191,9 +224,12 @@ def client_add(
         application_uri: str,
         public_key_secret_name: Optional[str] = None,
         private_key_secret_name: Optional[str] = None,
+        overwrite_secret: Optional[bool] = False,
     ) -> dict:
         # inform user if the provided cert was issued by a CA, the CA cert must be added to the issuers list.
-        logger.warning("Please ensure the certificate must be added to the issuers list if it was issued by a CA. ")
+        print(
```

Comment:
having a print is going to make the output of the command (assuming the output is meant to be tool friendly) harder to parse.

Reply:
For now the warning is safer because someone can disable it.